common/include/pcl/experimental/executor/cuda_executor.hpp

```diff
@@ -60,9 +67,14 @@ struct cuda_executor {
     return !operator==(lhs, rhs);
   }
 
-  template <typename F>
+  /**
+   * \brief  Launches the a single instance of the kernel
+   *
+   * \param f a CUDA kernel
+   */
+  template <typename Function>
   void
-  execute(F& f) const
+  execute(Function& f) const
```

Comment:
Functor instead of function (in other places too)

Reply:
`Callable` might be a better name I think. Will leave this for post-GSoC